README.md

```diff
@@ -66,10 +66,9 @@ git clone https://github.com/ethereum/remix-project.git
 * Build `remix-project`:
 ```bash
 cd remix-project
-yarn install
-yarn run build:libs // Build remix libs
-nx build
-nx serve
+yarn
+yarn run build
```

Comment:
'yarn build' builds only the remix-ide project and its dependencies, the libs project build is not part of that. libs are treated as source files just like the remix-ui.
'yarn build:libs' builds all libs seperately

If you are serving the ide, you don't need to build it first, yarn serve will just do that too. The build script is mostly there for e2e.

you don't need to do 'yarn run', just the script name